src/blob/header.rs

```diff
@@ -60,4 +60,9 @@ impl Header {
         }
         Ok(())
     }
+
+    #[inline]
+    pub(crate) fn serialized_size(&self) -> bincode::Result<u64> {
+        serialized_size(&self)
```

Comment:
It is hard to imagine, that this can fail. Please, use `expect` and convert result to `usize`